### PR TITLE
created migration file, movie_likes

### PR DESCRIPTION
**Pull request by blu3skies**

### CODE REVIEW

File: db/migrate/20200404095742_movie_likes.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class MovieLikes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :movie_likes do |t|
+
+      #movies foreign_key
+      t.references :movie, foreign_key: true
+
+      #users foreign_key
+      t.references :user, foreign_key: true
+
```
Comment (jasylwong) [inline db/migrate/20200404095742_movie_likes.rb:10]: t.timestamps

File: db/migrate/20200404095742_movie_likes.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class MovieLikes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :movie_likes do |t|
+
+      #movies foreign_key
```
Comment (jasylwong) [inline db/migrate/20200404095742_movie_likes.rb]: Rubocop: Add a space after the hashtag

File: db/migrate/20200404095742_movie_likes.rb (hunk 1 above)
Comment (jasylwong) [inline db/migrate/20200404095742_movie_likes.rb:10]: Rubocop: Add timestamps when creating a new table
ie add `t.timestamps` after the t.references lines

File: db/migrate/20200404095742_movie_likes.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      t.references :movie, foreign_key: true
+
+      #users foreign_key
+      t.references :user, foreign_key: true
```
Comment (jasylwong) [inline db/migrate/20200404095742_movie_likes.rb:9]: Rubocop: Add a space after the hashtag

**Review by jasylwong — CHANGES_REQUESTED** [review]

**Review by elliebjones — APPROVED** [review]
Looking good!

**Review by jasylwong — APPROVED** [review]

**Pull request merged**